### ISSUE TITLE
From LVTF: WCAG/APCA compare

### ISSUE THREAD

**Issue by Myndex** [Myndex/SAPC-APCA#54]
As I read through this thread, particularly one titled "shootout", there's two aspects I'm not getting that I hope someone can help with. These come from various experiences from usability testing, where I've seen quite a variety in which colour combinations cause issues for various people. I get uncomfortable when the visibility of a colour combination to people without an impairment is used as evidence for it's visibility from an accessibility point of view.

The two main questions I have are:

1. When talking about percentages of colour combinations that aren't right with a particular model, is that out of the whole colour space of combinations, or some sub-set?
2. Is it in comparison to a theoretical model, or the variety of human sight?

I think I need to show my current mental modal about this. I am assuming that if you looked at a colour space of hues, and mapped which people (with no visual impairment) could read against white text, it might look something like this:
<img src="https://user-images.githubusercontent.com/216630/148929517-20c26d9e-29d1-4e10-b200-b528ea18330f.png" width="400" alt="A rectangle containing a rainbow of colours horizontally, with deep colours at the top fading to white at the bottom, with a dark line drawn across it almost halfway up.">

Where anything under the line didn't have enough contrast against a white background. That line would probably wave up and down a bit with different hues, but just take that as a baseline for regular vision.

Then, if someone had low-vision with less acuity, presumably that line needs to move up?

<img src="https://user-images.githubusercontent.com/216630/148930414-33ef117b-06f8-4fab-aef0-21989ab1e6dc.png" width="400" alt="The same rectangle with a dark line drawn across it just over halfway up.">

Then, if we accounted for various colour deficiencies (e.g. protanopia) you'd need to adjust that line in various ways depending on the hue used.
<img src="https://user-images.githubusercontent.com/216630/148931028-fc961514-a20d-404d-8ae9-0c273751a721.png" width="400" alt="The same rectangle with a wavy line going further down with red, further up with blue.">

Then, to produce an overall model that accounted for both you'd want to be failing anything under either line:
<img src="https://user-images.githubusercontent.com/216630/148932089-0f5657bc-6da1-4560-b526-a655b42c8781.png" width="400" alt="The same rectangle with both the straight and the wavy line included, they overlap.">

I'm also assuming that the WCAG 2 formula is similar to the straight line, and the formula behind APCA is more bendy (apologies for the technical language ;-) ) to better match (non-impaired?) perception. 

However, when talking about percentages, what is it if you include all the colours that would fail both? Because (from my mental model above) there are a ton of combinations that are terrible and should fail both, so a ~50% incorrect rate just doesn't sound right.

_Originally posted by @alastc in https://github.com/w3c/low-vision-a11y-tf/discussions/131#discussioncomment-1946983_
